src/shapes/Circle.ts

```diff
@@ -19,20 +19,25 @@ interface UniqueCircleProps {
   radius: number;
 
   /**
-   * degrees of start of the circle.
-   * probably will change to degrees in next major version
+   * Angle for the start of the circle, in degrees.
    * @type Number 0 - 359
    * @default 0
    */
   startAngle: number;
 
   /**
-   * End angle of the circle
-   * probably will change to degrees in next major version
+   * Angle for the end of the circle, in degrees
    * @type Number 1 - 360
    * @default 360
    */
   endAngle: number;
+
+  /**
+   * Orientation for the direction of the circle.
+   * Setting to false will switch the arc of the circle to traverse from startAngle to endAngle in a counterclockwise direction,
+   * @default true
+   */
+  clockwise: boolean;
```

Comment:
Look at MDN ref: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/arc

I wish we align to it

Reply:
I totally agree with you. It's consistent, convenient, and conventional. It should be called counterclockwise.